Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -140,6 +144,11 @@ class BrowserViewController: UIViewController {
         self.readerModeCache = ReaderMode.cache(for: tabManager.selectedTab)
         self.crashedLastSession = crashedLastSession
         self.safeBrowsing = safeBrowsingManager
+        
+        #if !NO_REWARDS
+        rewards = BraveRewards(configuration: .default)
```

Comment:
Not sure that we care right now, but we will want to use `.production` configuration for `AppChannel.isRelease`

Reply:
Defaults means staging env?

cc @jhreis to decide on that, I think we should wait with .production setup for now

Reply:
Yeah, default is staging env